Resfeber/Model Controller/ProfileController.swift

```diff
@@ -349,16 +347,17 @@ class ProfileController {
                 NSLog("Error fetching image for url: \(url.absoluteString), error: \(error)")
                 return
             }
-            
+
             guard let data = data,
-                let image = UIImage(data: data) else {
-                    return
+                  let image = UIImage(data: data)
+            else {
+                return
             }
```

Comment:
Consider alternative formatting for the above `guard` statement:
```
guard let data = data, let image = UIImage(data: data) else { return }
```

Reply:
Fixed